rossiiskii-fiziologicheskii-zhurnal-imeni-i-m-sechenova.csl

```diff
@@ -0,0 +1,172 @@
+    </terms>
+  </locale>
+  <macro name="editor-translator">
+    <names variable="editor translator" delimiter="; ">
+      <label form="verb-short" suffix=" "/>
+      <name name-as-sort-order="all" sort-separator=" " initialize-with="." delimiter=", " delimiter-precedes-last="always"/>
+    </names>
+  </macro>
+  <macro name="author">
+    <names variable="author">
+      <name name-as-sort-order="all" sort-separator=" " initialize-with="." delimiter=", " delimiter-precedes-last="always"/>
+    </names>
+  </macro>
+  <macro name="title">
+    <choose>
+      <if type="webpage">
+        <text variable="title" suffix=" "/>
+        <text term="online" text-case="sentence" prefix="[" suffix="]"/>
+      </if>
+      <else>
+        <text variable="title"/>
+      </else>
+    </choose>
+  </macro>
+  <macro name="edition">
+    <choose>
+      <if is-numeric="edition">
+        <group delimiter=" ">
+          <number variable="edition" form="ordinal"/>
+          <text term="edition" form="short"/>
+        </group>
+      </if>
+      <else>
+        <text variable="edition"/>
+      </else>
+    </choose>
+  </macro>
+  <macro name="type-number">
+    <choose>
+      <if type="patent">
+        <text term="letter" form="short"/>
+        <text variable="number" prefix=" " suffix=" USA"/>
+      </if>
+      <else>
+        <group delimiter=" ">
+          <text variable="genre"/>
+          <text variable="number"/>
+        </group>
+      </else>
+    </choose>
+  </macro>
+  <macro name="publisher">
+    <group delimiter=". ">
+      <text variable="publisher-place"/>
+      <text variable="publisher"/>
+    </group>
+  </macro>
+  <macro name="issued-year">
+    <date variable="issued">
+      <date-part name="year"/>
+    </date>
+  </macro>
+  <macro name="locators">
+    <group delimiter=" ">
+      <group delimiter=" ">
+        <text variable="volume"/>
+      </group>
+      <group delimiter=" " prefix="(" suffix=")">
+        <text variable="issue"/>
+      </group>
+    </group>
+  </macro>
+  <macro name="page">
+    <group delimiter=" ">
+      <text variable="page"/>
+    </group>
+  </macro>
+  <macro name="access">
+    <choose>
+      <if type="webpage">
+        <choose>
+          <if variable="URL">
+            <text variable="URL" prefix="URL: "/>
+            <group prefix=" (" suffix=")" delimiter=": ">
+              <text term="accessed"/>
+              <date variable="accessed">
+                <date-part name="day" suffix="." form="numeric-leading-zeros"/>
+                <date-part name="month" suffix="." form="numeric-leading-zeros"/>
+                <date-part name="year"/>
+              </date>
+            </group>
+          </if>
+        </choose>
+      </if>
+    </choose>
+  </macro>
+  <citation collapse="citation-number">
+    <sort>
+      <key variable="citation-number"/>
+    </sort>
+    <layout vertical-align="baseline" delimiter=", " prefix="[" suffix="]">
```

Comment:
change ```vertical-align="baseline"``` to ```vertical-align="sup"``` here. Make sure you still want the [ ] around the superscript, otherwise delete prefix and suffix here.